src/sass/common.scss

```diff
@@ -331,15 +331,6 @@ fieldset .error-msg {
     position: absolute;
     top: -9999px;
 }
-.indicate-loading {
```

Comment:
for this we need to remove from js also src/javascript/binary/pages/bet/bet_form.js